features/transform-box.yml

```diff
@@ -0,0 +1,10 @@
+name: transform-box
+description: The `transform-box` CSS property sets the reference box for transformations.
```

Comment:
Ditto, I like the spec language, which gives just a bit more colour: "relative to the position and dimensions of the reference box". (The MDN description I found oddly phrased.)

Reply:
"Oddly phrased" seems to be the standard when looking at descriptions of this property. I updated it, and hope this is one standard I didn't match, but the phrasing is tricky.